pkg/trait/logging.go

```diff
@@ -73,7 +71,6 @@ func (l loggingTrait) Apply(environment *Environment) error {
 			if environment.Integration.Status.Dependencies == nil {
 				environment.Integration.Status.Dependencies = make([]string, 0)
 			}
-			util.StringSliceUniqueAdd(&environment.Integration.Status.Dependencies, depQuarkusLoggingJson)
```

Comment:
It seems like the whole code block can be removed?

Reply:
Thanks! Yes it can be removed.